sources/Applications/Web/Angular/Apps.Web.Angular.csproj

```diff
@@ -38,8 +39,7 @@
   <Target Name="PublishRunWebpack" AfterTargets="ComputeFilesToPublish">
```

Comment:
Add a condition that indicates the build configuration as debug/release in this way:
<Target Name="PublishRunWebpack" AfterTargets="ComputeFilesToPublish" Condition="'$(Configuration)'=='Release'">